<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1307</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Grimmo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="pages">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>1291</width>
      <height>741</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="login">
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>1281</width>
        <height>721</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>450</number>
       </property>
       <property name="topMargin">
        <number>150</number>
       </property>
       <property name="rightMargin">
        <number>450</number>
       </property>
       <property name="bottomMargin">
        <number>300</number>
       </property>
       <item>
        <widget class="QLabel" name="title">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Connection</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="username_login">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Nom Prenom</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="password_login">
         <property name="placeholderText">
          <string>Mot de passe</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="connect_login">
         <property name="text">
          <string>Se connecter</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="home">
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>1281</width>
        <height>611</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QScrollArea" name="scrollArea_2">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>636</width>
            <height>607</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QCalendarWidget" name="calendarWidget"/>
         </item>
         <item>
          <widget class="QScrollArea" name="scrollArea">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents_3">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>633</width>
              <height>299</height>
             </rect>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="add_bien">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>180</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>rue</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>230</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>code post</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="add_ville_add_bien">
      <property name="geometry">
       <rect>
        <x>482</x>
        <y>150</y>
        <width>321</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="add_rue_add_bien">
      <property name="geometry">
       <rect>
        <x>482</x>
        <y>200</y>
        <width>321</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="add_postal_add_bien">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>250</y>
        <width>321</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>290</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>type</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>350</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>surface</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>410</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>nb piece</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="add_surface_add_bien">
      <property name="geometry">
       <rect>
        <x>482</x>
        <y>370</y>
        <width>321</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>120</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ville</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="location_add_bien_btnradio">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>490</y>
        <width>95</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>location</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="acheter_add_bien_btnradio">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>490</y>
        <width>95</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>acheter</string>
      </property>
     </widget>
     <widget class="QComboBox" name="add_type_add_bien">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>310</y>
        <width>321</width>
        <height>22</height>
       </rect>
      </property>
      <property name="currentText">
       <string/>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>481</x>
        <y>440</y>
        <width>321</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
     </widget>
     <widget class="QPushButton" name="btnajouter_add_bien">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>680</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="price_entry">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>530</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="mois_label">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>550</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>/mois</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="agenda_2">
     <widget class="QWidget" name="horizontalLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>771</width>
        <height>641</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QCalendarWidget" name="calendarWidget_2"/>
       </item>
      </layout>
     </widget>
     <widget class="QTabWidget" name="btn_add_event">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>100</y>
        <width>481</width>
        <height>511</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>401</width>
          <height>631</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QScrollArea" name="scrollArea_3">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents_4">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>379</width>
              <height>614</height>
             </rect>
            </property>
            <widget class="QLabel" name="label_16">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>20</y>
               <width>35</width>
               <height>10</height>
              </rect>
             </property>
             <property name="text">
              <string>titre</string>
             </property>
            </widget>
            <widget class="QTextBrowser" name="textBrowser">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>30</y>
               <width>256</width>
               <height>61</height>
              </rect>
             </property>
            </widget>
            <widget class="QLabel" name="label_17">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>110</y>
               <width>35</width>
               <height>10</height>
              </rect>
             </property>
             <property name="text">
              <string>rue</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_18">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>130</y>
               <width>41</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>ville</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_19">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>150</y>
               <width>51</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>code postale</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_20">
             <property name="geometry">
              <rect>
               <x>160</x>
               <y>110</y>
               <width>35</width>
               <height>10</height>
              </rect>
             </property>
             <property name="text">
              <string>date</string>
             </property>
            </widget>
            <widget class="QLabel" name="label_21">
             <property name="geometry">
              <rect>
               <x>160</x>
               <y>140</y>
               <width>35</width>
               <height>10</height>
              </rect>
             </property>
             <property name="text">
              <string>heure</string>
             </property>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>titre</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="add_title_event">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>40</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="add_rue_event">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>170</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="add_cpostal_event">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>210</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="add_ville_event">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>250</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextEdit" name="add_description_event">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>80</y>
          <width>104</width>
          <height>64</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>70</y>
          <width>35</width>
          <height>10</height>
         </rect>
        </property>
        <property name="text">
         <string>description</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>160</y>
          <width>35</width>
          <height>10</height>
         </rect>
        </property>
        <property name="text">
         <string>rue</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>200</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>code postal</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>240</y>
          <width>35</width>
          <height>10</height>
         </rect>
        </property>
        <property name="text">
         <string>ville</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="dateEdit_event">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>290</y>
          <width>110</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QTimeEdit" name="timeEdit_event">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>350</y>
          <width>118</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>280</y>
          <width>35</width>
          <height>10</height>
         </rect>
        </property>
        <property name="text">
         <string>date</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>320</y>
          <width>35</width>
          <height>10</height>
         </rect>
        </property>
        <property name="text">
         <string>heure</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_add_event_2">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>370</y>
          <width>56</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>ajouter</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="view_bien">
     <widget class="QWidget" name="verticalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>511</width>
        <height>621</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QScrollArea" name="liste_view_bien">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>489</width>
            <height>604</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>100</y>
        <width>741</width>
        <height>621</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>MAP </string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="add_agent">
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>1281</width>
        <height>721</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>450</number>
       </property>
       <property name="topMargin">
        <number>200</number>
       </property>
       <property name="rightMargin">
        <number>450</number>
       </property>
       <property name="bottomMargin">
        <number>200</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Ajouter un utilisteur</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="last_name_add_user">
         <property name="placeholderText">
          <string>Nom</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="first_name_add_user">
         <property name="placeholderText">
          <string>Prénom</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="phone_add_user">
         <property name="placeholderText">
          <string>Téléphone</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="password_add_user">
         <property name="placeholderText">
          <string>Mot de passe</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="groups_add_user">
         <item>
          <property name="text">
           <string>Agent</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Responsable</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="add_user_btn">
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="top_menu">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1281</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="menu">
     <item>
      <widget class="QLabel" name="username">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="disconnect">
       <property name="text">
        <string>Se déconnecter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="home_home">
       <property name="text">
        <string>Accueuil</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="add_user_home">
       <property name="text">
        <string>Ajouter un utilisateur</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="view_home">
       <property name="text">
        <string>Voir les biens</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="agenda">
       <property name="text">
        <string>Agenda</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="add_good">
       <property name="text">
        <string>Ajouter un bien</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
